drivers/spi/spi_ll_stm32.c

```diff
@@ -489,10 +489,18 @@ static int spi_stm32_configure(const struct device *dev,
 #endif
 }
 
```

Comment:
Formatting is a bit off at lines:
- `494`
- `500`
- `872`

Reply:
Suggestion?

Reply:
494:
Added <kbd>Tab</kbd> and 2x <kbd>Space</kbd>.
```c
		if (clock_control_get_rate(DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE),
					  (clock_control_subsys_t) &cfg->pclken[1], &clock) < 0) {
			LOG_ERR("Failed call clock_control_get_rate(pclk[1])");
			return -EIO;
		}
```

500:
Added <kbd>Tab</kbd> and 2x <kbd>Space</kbd>.
```c
		if (clock_control_get_rate(DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE),
					  (clock_control_subsys_t) &cfg->pclken[0], &clock) < 0) {
			LOG_ERR("Failed call clock_control_get_rate(pclk[0])");
			return -EIO;
		}
```

872:
Removed 3x <kbd>Space</kbd>.
```c
	if (clock_control_on(DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE),
			    (clock_control_subsys_t) &cfg->pclken[0]) != 0) {
		LOG_ERR("Could not enable SPI clock");
		return -EIO;
	}
```

Reply:
Done